src/ui/marker.js

```diff
@@ -127,11 +154,31 @@ class Marker {
             svg.appendChild(page1);
 
             element.appendChild(svg);
+
+            // if no element and no offset option given apply an offset for the default marker
```

Comment:
best to rebase from master so these changes are brought in that way. that should also resolve the merge conflicts

Reply:
sure thing, and apologies for consuming so much time with this, i feel like i've learned a lot going through this process if it's any consolation 🌀  👍

Reply:
sure thing, and apologies for taking so much time with this, i think